### PR TITLE
Created scripts to fix lint errors; fixed lint errors

### PR DESCRIPTION
**Pull request by mayavera**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,12 @@
     "watch": "npm run build -- --watch",
     "start": "webpack-dev-server --config examples/using-module/webpack.config.babel.js --context examples/using-module --open",
     "analyze": "webpack --profile --json --progress > stats.json",
-    "lint": "eslint . --cache && stylelint **/*.scss --cache"
+    "lint-js": "eslint . --cache",
+    "lint-sass": "stylelint **/*.scss --cache",
```
Comment (josephspens) [inline package.json]: I'm pretty sure the only sass is in one of your other PRs, not in the app yet

File: package.json [hunk 2]
```diff
@@ -14,7 +14,12 @@
...
+    "lint": "yarn lint-js && yarn lint-sass",
+    "fix-js": "yarn lint-js --fix",
```
Comment (josephspens) [inline package.json]: When would we want to run these commands? Is this for anyone who doesn't have the VSCode plugin?

File: package.json (hunk 1 above)
Comment (mayavera) [reply package.json]: Good point!

File: package.json (hunk 2 above)
Comment (mayavera) [reply package.json]: I run this from time to time. Sometimes my linter plugin acts up and I'd rather finish a feature, run this command, then figure out what's wrong with my tooling while my PR is being reviewed. It happens more than I'd like, but it's nice to have a quick backup to stay in the flow.

**Pull request merged**